test/uk/gov/hmrc/thirdpartyapplication/testutils/LogSuppressing.scala

```diff
@@ -24,7 +24,7 @@ import play.api.LoggerLike
 
 import scala.collection.mutable
 
-import scala.collection.JavaConversions._
+import scala.collection.JavaConverters._
 
```

Comment:
Is this import still needed?

Reply:
Yes.  Conversions was the old implicit variety.  Converters is the explicit version that allows .asScala merhods.

Reply:
Ah sorry thought they were the same. My mistake!